platforms/hyperledger-fabric/configuration/deploy-network.yaml

```diff
@@ -134,11 +134,13 @@
         config_file: "./build/configtx.yaml"
 
     # This role generate genesis block and channeltx
-    - include_role:
+    - name: "Create channel artifacts"
+      include_role:
         name: "create/channel_artifacts"
       vars:
         build_path: "./build"
         genesis: "{{ item.genesis }}"
+        channel_orderer_name: "{{ item.orderer.name | lower}}-net"
```

Comment:
Where is this variable used?

Reply:
This was previously used in the vault path, but not anymore. I'll remove this

Reply:
Removed in [ad25b2c](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1053/commits/ad25b2c131b4539efa614b1f5f263c972b7bf629)